cuda_core/tests/test_event.py

```diff
@@ -80,3 +80,63 @@ def test_is_done(init_cuda):
     # Without a sync, the captured work might not have yet completed
     # Therefore this check should never raise an exception
     assert event.is_done in (True, False)
+
+
+def test_error_timing_disabled():
+    device = Device()
+    device.set_current()
+    enabled = EventOptions(enable_timing=True)
+    disabled = EventOptions(enable_timing=False)
+    stream = device.create_stream()
+
+    event1 = stream.record(options=enabled)
+    event2 = stream.record(options=disabled)
+    stream.sync()
+    with pytest.raises(RuntimeError, match="^Both Events must be created with timing enabled"):
+        event2 - event1
+
+    event1 = stream.record(options=disabled)
+    event2 = stream.record(options=disabled)
+    stream.sync()
+    with pytest.raises(RuntimeError, match="^Both Events must be created with timing enabled"):
+        event2 - event1
+
+    event1 = stream.record(options=enabled)
+    event2 = stream.record(options=enabled)
+    stream.sync()
+    event2 - event1
```

Comment:
I'd delete this last block, because that's already covered more comprehensively under the existing `test_timing` (`test_timing_success`).

Reply:
d66580e64ad13c92b9deff4c2eff1a453ca8f8ab

Reply:
Did you maybe forget to push the commits?

Reply:
No. I'm still working on addressing feedback and would prefer to push the commits all at once instead of triggering the CI multiple times.

Reply:
Our CI is manual trigger only, so don't worry about excessive pushes 🙂